diff --git a/libraries/ConfigurableFirmata/src/utility/Boards.h b/libraries/ConfigurableFirmata/src/utility/Boards.h
index 722c36d..77fc770 100644
--- a/libraries/ConfigurableFirmata/src/utility/Boards.h
+++ b/libraries/ConfigurableFirmata/src/utility/Boards.h
@@ -142,7 +142,7 @@ writePort(port, value, bitmask):  Write an 8 bit port.
 
 // Arduino Duemilanove, Diecimila, Uno, Nano, etc.
 #if defined(__AVR_ATmega168__) || defined(__AVR_ATmega328P__) || defined(__AVR_ATmega328__)
-#if defined(NUM_ANALOG_INPUTS) && NUM_ANALOG_INPUTS == 6
+#if defined(NUM_ANALOG_INPUTS) && NUM_ANALOG_INPUTS == 6 && !defined(ARDUINO_UNO_ULTRA)
 #define TOTAL_ANALOG_PINS       6
 #define TOTAL_PINS              20 // 14 digital + 6 analog
 #else
